csharp/tutorials/nullable-reference-migration/finished/SimpleFeedReader.Tests/Services/NewsServiceTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using AutoMapper;
+using SimpleFeedReader.Services;
+using SimpleFeedReader.ViewModels;
+using Xunit;
+
+namespace SimpleFeedReader.Tests.Services
+{
+#nullable enable
```

Comment:
I'd expect that within a genuinely *complete* migration, nullability would be enabled at a project level instead.

I appreciate that it's a lot of work to maintain three versions, but I wonder whether a model of before/during/after might be useful. I would imagine it would be *particularly* helpful to look at specific examples of what happens when you've migrated one piece of code but not others... and give some guidance about what to migrate first.

Reply:
Good points. I'm going to defer these for the moment. I am sure that I will create a longer form tutorial that encompasses multiple projects (including test projects) as the release gets closer.

I need to migrate a few more codebases myself before I feel confident that I'm providing solid guidance on the strategy. (And I want to gather input from others that have migrated code).